pkg/nodediscovery/cell.go

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package nodediscovery
+
+import "github.com/cilium/cilium/pkg/hive/cell"
+
+var Cell = cell.Module(
```

Comment:
Let's document the different things NodeDiscovery does with a comment here. E.g. an expanded version of what we have in daemon/cmd/cells.go.

Reply:
I meant documenting `var Cell`, e.g. like in `pkg/endpointmanager/cell.go`. Use-case is when hovering over `nodediscovery.Cell` in `daemon/cmd/cells.go` we'd get a bit more detailed description of what objects are provided from here and what they do. Sorry for the nit-picking. Trying to make the habit stick :)

Reply:
Right, I had misunderstood what you meant by "what we have in daemon/cmd/cells.go"